packages/datadog_flutter_plugin/android/src/main/kotlin/com/datadoghq/flutter/DatadogRumPlugin.kt

```diff
@@ -192,6 +193,16 @@ class DatadogRumPlugin internal constructor(
         result.success(null)
     }
 
+    private fun getCurrentSessionId(call: MethodCall, result: MethodChannel.Result) {
+        rum?.let {
+            it.getCurrentSessionId { sessionId ->
+                result.success(sessionId)
+            }
+        } ?: run {
```

Comment:
what is the reason to use `run` here? I think it is possible to write just `?: result.invalidOperation("RUM is not enabled")`

Reply:
I actually don't need to call the result at all, as `onMethodCall` already checks for the existence of `rum`.